src/NHibernate.Test/Linq/ParameterTypeLocatorTests.cs

```diff
@@ -0,0 +1,427 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Linq.Expressions;
+using NHibernate.DomainModel.Northwind.Entities;
+using NHibernate.Engine.Query;
+using NHibernate.Linq;
+using NHibernate.Linq.Visitors;
+using NHibernate.Type;
+using NUnit.Framework;
+using Remotion.Linq.Clauses;
+
+namespace NHibernate.Test.Linq
+{
+	public class ParameterTypeLocatorTests : LinqTestCase
```

Comment:
Builds on master are crashed by this file, which for some unknown reason is currently polluting master builds on the TeamCity VM.

Reply:
Never mind, it was not on master, forget about it, I am going to delete these comments.